docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/refget/refget.md

```diff
@@ -0,0 +1,246 @@
+
+### Use of multiple identifiers for a sequence
+
+Refget's specification details two checksum algorithms which can be used to generate the identifier. These are a hex encoded `MD5` or a scheme called [`sha512t24u`](https://samtools.github.io/hts-specs/refget.html#refget-checksum-algorithm). A GA4GH identifier is generated by adding `ga4gh:SQ.` to a `sh512t24u` checksum e.g. `ga4gh:SQ.0qw_sn8Cl7OmMTFlukjFD2DUejW0T80Y` and `17dab79b963ccd8e7377cef59a54fe1c` represent the same sequence.
+
+## Third party refget clients
+
+You can use the [Python `refget`](https://pypi.org/project/refget/) library to fetch sequences from the Ensembl refget instance.
+
+## Accessing sequence data for features in Ensembl
+
+The ["Getting started with Graphql"](../tools-and-apis/graphql/using-graphql.md) and ["Using Graphql"](../tools-and-apis/graphql/using-graphql.md) pages cover how to access data through Ensembl GraphQL service and the example below assumes that this service is being used.
+
+Before starting you must obtain a `genome_id` for the genome (assembly + annotation) of interest. We are using `"GCA_000001405.29`, which represents the human assembly GRCh38.p14 (hg38).
+
+```text
+query {
+  genomes(
+    by_assembly_accession_id: {assembly_accession_id: "GCA_000001405.29"})
+  {
+    genome_id
+  }
+}
+```
+
+returns:
+
+```json
+{
+  "data": {
+    "genomes": [
+      {
+        "genome_id": "a7335667-93e7-11ec-a39d-005056b38ce3"
+      }
+    ]
+  }
+}
+```
+
+### Fetching the genomic sequence for a feature
+
+Using the previously retrieved `genome_id`, we can search for a feature (such as a gene) you myst find the checksum for a feature's (gene, transcript etc.) region, and its start and end coordinates via `slice`
+
+```text
+query {
+  genes(
+    by_symbol: {genome_id: "a7335667-93e7-11ec-a39d-005056b38ce3", symbol: "JAG1"}
+  ) {
+    slice{
+      region{
+        sequence{
+          checksum
+        }
+      }
+      location{
+        start
+        end
+      }
+    }
+  
+  }
+}
+```
+
+returns:
+
+```json
+{
+  "data": {
+    "genes": [
+      {
+        "slice": {
+          "region": {
+            "sequence": {
+              "checksum": "b18e6c531b0bd70e949a7fc20859cb01"
+            }
+          },
+          "location": {
+            "start": 10637684,
+            "end": 10673999
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+Then use the information from the previous step to query refget (Curl for clarity). Note you must convert from Ensembl coordinates (1-based) to 0-based coordinates.
+
+```bash
+curl -H'Accept: text/plain' "https://beta.ensembl.org/data/refget/sequence/b18e6c531b0bd70e949a7fc20859cb01?start=10637683&end=10673999"
+
+```
+
+returns:
+
+```text
+GGCCATTAATCCAGTGGTGTTTATT ... CCCGGAGAGCCCGTCT% 
```

Comment:
the percent sign in the end is an artifact of the terminal

Reply:
fixed